src/uct/tcp/tcp_ep.c

```diff
@@ -495,24 +496,8 @@ static inline void
 uct_tcp_ep_zcopy_ctx_consume_iov(uct_tcp_ep_zcopy_ctx_t *zcopy_ctx,
                                  size_t consumed)
 {
-    struct iovec *iov = zcopy_ctx->iov;
-    size_t i;
-
-    ucs_assert(zcopy_ctx->iov_index  <= zcopy_ctx->iov_cnt);
-
-    for (i = zcopy_ctx->iov_index; i < zcopy_ctx->iov_cnt; i++) {
-        if (consumed < iov[i].iov_len) {
-            iov[i].iov_len  -= consumed;
-            iov[i].iov_base  = UCS_PTR_BYTE_OFFSET(iov[i].iov_base,
-                                                   consumed);
-            zcopy_ctx->iov_index   = i;
-            return;
-        }
-
-        consumed -= iov[i].iov_len;
-    }
-
-    ucs_assert(!consumed && (i == zcopy_ctx->iov_index));
+    ucs_sys_consume_iov(zcopy_ctx->iov, zcopy_ctx->iov_cnt,
```

Comment:
i don't think we need the wrapper function uct_tcp_ep_zcopy_ctx_consume_iov anymore?